lib/ansible/modules/infrastructure/stacki/stacki_host.py

```diff
@@ -0,0 +1,324 @@
+
+-
+# Remove a host named test-1
+- stacki_host: name=test-1 stacki_user=usr stacki_password=pwd stacki_endpoint=url state=absent
+'''
+
+RETURN = '''
+changed:
+  description: response to whether or not the api call completed successfully
+  returned: always
+  type: boolean
+  sample: true
+
+stdout:
+  description: the set of responses from the commands
+  returned: always
+  type: list
+  sample: ['...', '...']
+
+stdout_lines:
+  description: the value of stdout split into a list
+  returned: always
+  type: list
+  sample: [['...', '...'], ['...'], ['...']]
+'''
+
+import os
+import re
+import tempfile
+
+try:
+    import requests
+    HAS_REQUESTS = True
+except ImportError:
+    HAS_REQUESTS = False
+
+
+def stack_auth(module, result, params):
+
+    endpoint    = params['stacki_endpoint']
+    auth_creds  = {'USERNAME': params['stacki_user'],
+                   'PASSWORD': params['stacki_password']}
+
+    client = requests.session()
+    client.get(endpoint)
+
+    init_csrf = client.cookies['csrftoken']
+
+    header = {'csrftoken': init_csrf, 'X-CSRFToken': init_csrf,
+              'Content-type': 'application/x-www-form-urlencoded'}
+
+    login_endpoint = endpoint + "/login"
+
+    login_req = client.post(login_endpoint, data=auth_creds, headers=header)
+
+    csrftoken = login_req.cookies['csrftoken']
+    sessionid = login_req.cookies['sessionid']
+
+    auth_creds.update(CSRFTOKEN=csrftoken, SESSIONID=sessionid)
+
+    return client, auth_creds
+
+
+def stack_build_header(auth_creds):
+    header = {'csrftoken': auth_creds['CSRFTOKEN'],
+              'X-CSRFToken': auth_creds['CSRFTOKEN'],
+              'sessionid': auth_creds['SESSIONID'],
+              'Content-type': 'application/json'}
+
+    return header
+
+
+def stack_check_host(params, header, client):
+
+    hostname    = params['name']
+    endpoint    = params['stacki_endpoint']
+
+    stack_r = client.post(endpoint, data=json.dumps({"cmd": "list host"}),
+                          headers=header)
+    if hostname in stack_r.text:
+        return True
+    else:
+        return False
+
+
+def stack_sync(endpoint, header, client):
+
+    stack_r = client.post(endpoint, data=json.dumps({ "cmd": "sync config"}),
+                          headers=header)
+        
+    stack_r = client.post(endpoint, data=json.dumps({"cmd": "sync host config"}),
+                          headers=header)
+
+    rc = stack_r.status_code
+
+    if rc == 200:
+        rc = 0
+
+    return rc, stack_r.text
+
+
+def stack_force_install(result, params, header, client):
+
+    hostname = params['name']
+    endpoint = params['stacki_endpoint']
+    data = dict()
+    rc = None
+    changed = False
+
+    data['cmd'] = "set host boot {} action=install" \
+        .format(hostname)
+    stack_r = client.post(endpoint, data=json.dumps(data), headers=header)
+    rc = stack_r.status_code if stack_r.status_code != 200 else stack_r.status_code
```

Comment:
The Shippable CI failure is due to: 

```
2016-12-19 16:09:41 Run command: python2.4 -m compileall -fq ./lib/ansible/modules/infrastructure/stacki/stacki_host.py
2016-12-19 16:09:41 Compiling ./lib/ansible/modules/infrastructure/stacki/stacki_host.py ...
2016-12-19 16:09:41   File "./lib/ansible/modules/infrastructure/stacki/stacki_host.py", line 174
2016-12-19 16:09:41     rc = stack_r.status_code if stack_r.status_code != 200 else stack_r.status_code
2016-12-19 16:09:41                               ^
2016-12-19 16:09:41 SyntaxError: invalid syntax
```
This may also apply to line 210